documentation/puppetdb-faq.markdown

```diff
@@ -24,69 +25,68 @@ Yes. See [Migrating Data][migrating] for more information.
-forward the plaintext port or change the interface PuppetDB binds on to one
+forward the plain text port or change the interface PuppetDB binds on to one
 that is accessible from the outside world. In the latter case, you'll want to
 use some other means to secure PuppetDB (for instance, by restricting which
 hosts are allowed to talk to PuppetDB through a firewall).
 
-* You're trying to talk over SSL and nobody trusts anybody else
+* You're trying to talk over SSL and nobody trusts anybody else.
 
-Because PuppetDB uses the certificate authority of your Puppet
+Because PuppetDB uses the certificate authority (CA) of your Puppet
 infrastructure, and a certificate signed by it, PuppetDB doesn't trust your
 browser, and your browser doesn't trust PuppetDB. In this case, you'll need to
 give your browser a certificate signed by your Puppet CA. Support for client
 certificates is varied between browsers, so it's preferred to connect over
-plaintext, as outlined above.
+plain text, as outlined above.
 
 ## Does PuppetDB support Puppet apply?
 
-Yes. The setup is quite different to the normal master based setup, so
+Yes. However, the setup is quite different from the normal master-based setup, so
 [consult the documentation][connect_puppet_apply] for more details.
 
 ## Why is PuppetDB written in Java?
 
 Actually, PuppetDB isn't written in Java at all! It's written in a language
 called Clojure, which is a dialect of Lisp that runs on the Java Virtual
-Machine. Several other languages were prototyped, including Ruby and JRuby, but
-they lacked the necessary performance.  We chose to use a JVM language because
+Machine (JVM). Several other languages were prototyped, including Ruby and JRuby, but
+they lacked the necessary performance. We chose to use a JVM language because
 of its excellent libraries and high performance. Of the available JVM
 languages, we used Clojure because of its expressiveness, performance, and
-previous experience with the language on our team.
+our team's previous experience with the language.
 
 ## Which versions of Java are supported?
 
 The officially supported versions are OpenJDK and Oracle JDK 1.7 on
-Debian-derived distros and 1.8 on RHEL-derived distros. Other versions may work
-and issues will be addressed on a best effort basis, but support is not guaranteed.
+Debian-derived distros and 1.8 on RHEL-derived distros. Other versions may work,
+and issues will be addressed on a best-effort basis, but support is not guaranteed.
 
 ## Which databases are supported?
 
 PostgreSQL is the recommended database for production use. PuppetDB also ships
-with an embedded HyperSQL database which is suitable for very small or proof of
-concept deployments. As with our choice of language, we prototyped several
+with an embedded HyperSQL database, which is suitable for very small or proof-of-concept deployments. As with our choice of language, we prototyped several
```

Comment:
We should chop this after the first sentence, since HyperSQL is no longer supported in this branch.